aguasdearuanda/zelio
# Remove Public Justification Form

### Description

This PR removes the public justification form.

### Changelog
- translate errors pages to Portuguese
- remove public justification controller
- remove public justification route
- remove public justification directory

### How to test
You must try to access the page `http://localhost:3000/justificativa/novo` and must-see an error message.
